script.js

```diff
@@ -0,0 +1,58 @@
+let books = [];
+
+const addBookButton = document.querySelector('button');
+addBookButton.onclick = function addBookButton() {
+  if (localStorage.getItem('savedArray') != null) {
+    books = JSON.parse(localStorage.getItem('savedArray'));
+  }
+  if (document.querySelector('.title').value !== '' && document.querySelector('.author').value !== '') {
+    const btitle = document.querySelector('.title').value;
+    const bauthor = document.querySelector('.author').value;
+    const book = {
+      title: btitle,
+      author: bauthor,
+    };
+    books.push(book);
+  }
+
+  localStorage.setItem('savedArray', JSON.stringify(books));
+};
+
+function showBooks() {
+  if (localStorage.getItem('savedArray') != null) {
+    books = JSON.parse(localStorage.getItem('savedArray'));
+  }
+  const contain = document.querySelector('.books');
+  for (let i = 0; i < books.length; i += 1) {
+    const btitle = document.createElement('h5');
+    btitle.className = 'bookname';
+    btitle.id = books[i].title;
+    btitle.textContent = books[i].title;
+    const bauthor = document.createElement('p');
+    bauthor.className = 'bookauthor';
+    bauthor.textContent = books[i].author;
+    btitle.appendChild(bauthor);
+    const removeButton = document.createElement('button');
+    removeButton.setAttribute('onclick', `removeFunction('${books[i].title}')`);
+    removeButton.id = `remove${books[i].title}`;
+    removeButton.textContent = 'remove';
+    contain.appendChild(btitle);
+    contain.appendChild(removeButton);
+  }
+}
+
+showBooks();
+
+// eslint-disable-next-line no-unused-vars
+function removeFunction(title) {
+  for (let i = 0; i < books.length; i += 1) {
+    if (books[i].title === title) {
+      books.splice(i, 1);
```

Comment:
* This is an OPTIONAL suggestion as it is not a requirement for this project but i think in your spare time you can work on this.
* Using the array method `splice` might be expensive as it mutates the original array which may lead to unexpected and hard-to-debug issues [Read more on why it is not advisable](https://blog.sapegin.me/all/avoid-mutation/)